src/main/java/uk/ac/ebi/tsc/portal/api/team/controller/TeamRestController.java

```diff
@@ -683,6 +683,46 @@ public TeamResource getTeamByName(HttpServletRequest request, Principal principa
 		return new ResponseEntity<>("User  request was successfully sent to team owner " + teamResource.getName(), HttpStatus.OK);
 	}
 
+	@RequestMapping(value="/contact/email", method=RequestMethod.POST)
```

Comment:
This is also not the best design. You suggest that there are 2 resources: `contact` and `email`. Besides this method by the way it looks should be accepting new email and creating it as a resource, whereas it accepts the entire `team` and probably manipulates the entire list of emails.

Reply:
Modified paths, and made it a put request